data/DataType/string.yaml

```diff
@@ -0,0 +1,3 @@
+keyword: string
```

Comment:
there is no single most common 'string' , so this file should be called 'string_pascal.yaml' .  Worth nothing that string isnt even a basic data type of the Pascal language.  There are various compilers providing different definitions of what a string is for Pascal and its many derivatives.

Reply:
So should I remove it? 

I added it after this review comment: https://github.com/coala/coAST/pull/82#discussion_r238638338

Reply:
Renamed